src/components/App/App.js

```diff
@@ -31,24 +31,27 @@ export default class App extends React.Component {
       searchValue,
       onLogoClick,
       resultsValue,
-      loadingResults
+      loadingResults,
+      errorHandled
     };
   }
 
   onSearchChange = value => {
-    this.setState({ loadingResults: true });
+    this.setState({ searchValue : value, loadingResults: true });
     getRepos(value)
       .then(res => {
         this.setState({ searchValue : value, resultsValue : res , loadingResults: false });
       })
-      .catch(::console.error);
+      .catch(err => {
+        this.setState({ resultsValue: undefined, errorHandled: err, loadingResults: false });
```

Comment:
@giogonzo do you know if `this.setState({ x: undefined })` always works as expected?